en/services/mission.md

```diff
@@ -115,6 +115,9 @@ This section explains the main operations defined by the protocol.
 
 The diagram below shows the communication sequence to upload a mission to a drone (assuming all operations succeed).
 
+> **Note** Mission update must be robust! 
```

Comment:
Note, this is PX4 but not ArduPilot implementation for full mission upload. I have assumed the same for partial update. Will clarify for ArduPilot in the implemetation section.